articles/azure-functions/functions-app-settings.md

```diff
@@ -301,7 +301,7 @@ Only used when deploying to a Premium plan or to a Consumption plan running on W
 
 ## WEBSITE\_CONTENTOVERVNET
 
-For Premium plans only. A value of `1` enables your function app to scale when you have your storage account restricted to a virtual network. You should enable this setting when restricting your storage account to a virtual network. To learn more, see [Restrict your storage account to a virtual network](functions-networking-options.md#restrict-your-storage-account-to-a-virtual-network).
+For all Windows virtual network-supported SKUs in the Dedicated (App Service) plan and for Windows Elastic Premium plans. Consumption and Linux Elastic Premium plans aren't supported. A value of `1` enables your function app to scale when you have your storage account restricted to a virtual network. You should enable this setting when restricting your storage account to a virtual network. To learn more, see [Restrict your storage account to a virtual network](functions-networking-options.md#restrict-your-storage-account-to-a-virtual-network).
```

Comment:
We need to also fix the branding in the VNET article.
```suggestion
Supported for Windows on a Premium plan and Dedicated (App Service) plans (Standard and higher). Not supported for Linux Consumption and Premium plans. A value of `1` enables your function app to scale when you have your storage account restricted to a virtual network. You should enable this setting when restricting your storage account to a virtual network. To learn more, see [Restrict your storage account to a virtual network](functions-networking-options.md#restrict-your-storage-account-to-a-virtual-network).
```

Reply:
What exact changes are required here?